src/views/wallet/Validator.vue

```diff
@@ -54,6 +68,46 @@ export default class Validator extends Vue {
         clearInterval(this.intervalID)
     }
 
+    async evaluateCanRegisterNode() {
+        //This should evaluate 3 things:
+        //1. Is the user KYC verified
+        //2. Is the user a consortium member
+        //3. Is the node already registered
+        const result = await WalletHelper.getAddressState(this.addresses[0])
+        let addressStateKycVerified = 32
+        let addressStateConsortium = 34
+        let addressStateRegisterNode = 38
+        console.log(result)
+        let bits = new BN(result).toString(2)
+        console.log(bits)
+        if (!bits[bits.length - addressStateKycVerified - 1]) {
```

Comment:
Not sure how to properly read state here and I don't like this solution. @peak3d do you have any tips here?